travis/clone_oca_dependencies

```diff
@@ -60,6 +60,11 @@ def git_checkout(deps_checkout_dir, reponame, url, branch):
         command = ['git', 'clone', '-q', url, '-b', branch, checkout_dir]
         _logger.info('Calling %s', ' '.join(command))
         subprocess.check_call(command)
+    else:
+        command = ['git', '--git-dir=' + os.path.join(checkout_dir, '.git'),
```

Comment:
I'm not sure...
We are using this sentence since along time ago in runbot:
https://github.com/odoo/odoo-extra/blob/1e2d4bfdfe0e66e0a000e10edd8bf68b756eff46/runbot/runbot.py#L224

2015-12-01 9:35 GMT-05:00 Sylvain Garancher <notifications@github.com>:

> In travis/clone_oca_dependencies
> <https://github.com/OCA/maintainer-quality-tools/pull/300#discussion_r46284620>
> :
>
> > @@ -60,6 +60,11 @@ def git_checkout(deps_checkout_dir, reponame, url, branch):
> >          command = ['git', 'clone', '-q', url, '-b', branch, checkout_dir]
> >          _logger.info('Calling %s', ' '.join(command))
> >          subprocess.check_call(command)
> > +    else:
> > +        command = ['git', '--git-dir=' + os.path.join(checkout_dir, '.git'),
>
> When using the --git-dir argument, you should use --work-tree, to define
> the working directory of your git repository.
>
> Try calling git --git-dir=/some/path/.git from outside your git
> repository. The changes will be applied in your current working directory,
> instead of the git repository.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/maintainer-quality-tools/pull/300/files#r46284620>
> .
>



-- 
Moisés López Calderón
Vauxoo - OpenERP's Gold Partner
Mobile: (+521) 477-752-22-30
Office: (+52) 477-773-33-46
web: http://www.vauxoo.com
twitter: @vauxoo
           @moylop260
hangout: moylop260@vauxoo.com